les/peer.go

```diff
@@ -291,6 +291,11 @@ func (p *peer) updateCapacity(cap uint64) {
 	p.queueSend(func() { p.SendAnnounce(announceData{Update: kvList}) })
 }
 
+func (p *peer) responseID() uint64 {
```

Comment:
Do you need this as a separate function? It is only used in the handler so maybe it is more readable if you leave this part there too.